test/e2e/page-objects/pages/confirmations/redesign/transaction-confirmation.ts

```diff
@@ -6,92 +6,100 @@ import { RawLocator } from '../../../common';
 import Confirmation from './confirmation';
 
 class TransactionConfirmation extends Confirmation {
-  private walletInitiatedHeadingTitle: RawLocator;
+  private readonly advancedDetailsButton: RawLocator = `[data-testid="header-advanced-details-button"]`;
 
-  private dappInitiatedHeadingTitle: RawLocator;
```

Comment:
instead of declare the selectors here and then initialized them later, we can just inline-initialize them, to avoid having the double of the lines and make it more readable.
This way we also follow the same convention that we have in the other page object classes